mmv1/third_party/terraform/resources/resource_container_node_pool.go.erb

```diff
@@ -79,6 +79,49 @@ func resourceContainerNodePool() *schema.Resource {
 	}
 }
 
+var schemaBlueGreenSettings = &schema.Schema{
+	Type:     schema.TypeList,
+	Optional: true,
+	Computed: true,
```

Comment:
```suggestion
	Computed: true,
	MaxItems: 1,
	ConflictsWith: []string{"upgrade_settings.0.max_surge", "upgrade_settings.0.max_unavailable"},
```
Also propose to add `MaxItems: 1` here, as I believe it's not actually a list or array but a nested object.